packages/notifications/push/amplify_push_notifications_ios/ios/Classes/AmplifyPushNotificationsIosPlugin.swift

```diff
@@ -0,0 +1,90 @@
+import Flutter
```

Comment:
Let's empty this file or keep only the basic package structure so we can merge the iOS implementation afterwards and keep a clean commit history on this file.

Reply:
Yeah, emptied the class